pkg/neonvm/controllers/vm_qmp_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
-	"github.com/neondatabase/autoscaling/pkg/neonvm/controllers"
+	. "github.com/neondatabase/autoscaling/pkg/neonvm/controllers"
```

Comment:
suggestion: IIUC this change is unrelated? if so, let's remove it and the other changes to this file?

Reply:
reverted that change